service/controller/resource/instance/create_test.go

```diff
@@ -1,552 +0,0 @@
-package instance
```

Comment:
This test checks the functioning of instance selection.
This feature is not used by the workers anymore, so the test is unuseful.